packages/fx-core/src/core/index.ts

```diff
@@ -442,6 +445,9 @@ export class FxCore implements v3.ICore {
           }
         }
       }
+      if (ctx.projectSettings?.solutionSettings) {
```

Comment:
I think we need to check input capability to decide whether it's Azure hosting?

Reply:
I think in the future, we will ignore whether Azure hosting or not.

Reply:
Currently hosting type is used in extension to check what UI to show.